File: js/2.js
var sequence	= new Array(),
	sum			= 0;
sequence = [1, 2, 3]

while(sequence[sequence.length - 1] <= 4000000)
	sequence.push( sequence[sequence.length - 2] + sequence[sequence.length - 1] );

for(pos in sequence) {
	if(sequence[pos] % 2 == 0)
		sum += sequence[pos];
}

console.log(sum);

File: c++/2.cpp
#include <iostream>
using namespace std;

int main() {
	int fib[2] = {1, 2};
	int sum = 0;

	while(fib[1] < 4000000) {	
		if(fib[1] % 2 == 0) {
			sum += fib[1];
		}

		int tmp = fib[0];
		fib[0] = fib[1];
		fib[1] = fib[0] + tmp;
	}

	cout << sum << endl;
	return 0;
}
